src/shared/utilities/textUtilities.ts

```diff
@@ -29,8 +29,14 @@ export function getStringHash(text: string): string {
     return hash.digest('hex')
 }
 
+/**
+ * Using the 'sha1' hash is no longer recommended. This hash is used to specifically comply with the
+ * Spec: 'SSO Login Token Flow' and is used only to name a file. It is advised to use a more secure
```

Comment:
getSHA1StringHash is a general string utility. the Spec comment belongs at the caller

Reply:
done